src/plugins/plot/src/configuration/PlotSeries.js

```diff
@@ -360,6 +362,17 @@ define([
                 }
             }
 
+        },
+        /**
+         * Clears the plot series, unsubscribes and resubscribes
+         * @public
+         */
+        clearAndRefetch: function () {
+            if (this.unsubscribe) {
```

Comment:
This is called when the filters change, but I don't see where it actually updates the plot with the latest persisted filters? Indeed, when I change the filters from the inspector the plot filters do not appear to update.